src/Microsoft/MicrosoftUser.php

```diff
@@ -14,16 +14,21 @@ class MicrosoftUser extends User
     public function getAvatar()
     {
         $client = new Client();
-        $response = $client->get(
-            'https://graph.microsoft.com/v1.0/me/photo/$value',
-            [
-                RequestOptions::HEADERS => [
-                    'Accept'        => 'image/*',
-                    'Authorization' => 'Bearer '.$this->token,
-                ],
-            ]
-        );
 
-        return (new MicrosoftAvatar())->setResponse($response);
+        try {
+            $response = $client->get(
+                'https://graph.microsoft.com/v1.0/me/photo/$value',
+                [
+                    RequestOptions::HEADERS => [
+                        'Accept'        => 'image/*',
+                        'Authorization' => 'Bearer '.$this->token,
+                    ],
+                ]
+            );
+
+            return (new MicrosoftAvatar())->setResponse($response);
+        } catch (\GuzzleHttp\Exception\ClientException $e) {
```

Comment:
Can you import this `ClientException` ?

Reply:
@lucasmichot Updated to import `ClientException`